wandb/sdk/wandb_run.py

```diff
@@ -2219,8 +2231,21 @@ def _create_image_job(
         job_artifact = self._construct_job_artifact(
             name, source_info, installed_packages_list
         )
-        artifact = self.use_artifact(job_artifact)
-        return artifact
+
+        return job_artifact
+
+    def _log_job_artifact_with_image(self, docker_image_name: str) -> Artifact:
+        packages, in_types, out_types = self._make_job_source_reqs()
+        job_artifact = self._create_image_job(
+            in_types, out_types, packages, docker_image_name
+        )
+
+        artifact = self.log_artifact(job_artifact)
```

Comment:
This should be `use_artifact`

Reply:
Nevermind

Reply:
It should be log, since we want the first case of the call with use to be the actual run